.github/workflows/lint_test_build.yml

```diff
@@ -41,6 +41,22 @@ jobs:
         if: steps.verify-changed-files.outputs.files_changed == 'true'
         run: exit 1
 
+  documenation:
+    runs-on: ubuntu-22.04
+    needs: checkout
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/workflows/rafiki/env-setup
+      - name: generate graphql api documentation
+        run: pnpm --filter documentation docs:graphql:generate && pnpm format
```

Comment:
```suggestion
        run: pnpm --filter documentation docs:graphql:generate
```
Maybe?

Reply:
I believe we need the format. Without any gql changes, generating the docs will have changes due to not being formatted. Meaning the currently committed api docs have been formatted. This step would always fail because there would always be changes. Additionally, I think committing without formatting would cause the `pnpm check` cmd to fail:

https://github.com/interledger/rafiki/blob/d0a1a47daeb1467376dc97cc56029b57e475b95d/.github/workflows/lint_test_build.yml#L12-L19

Reply:
I don't think it affects `pnpm checks` since that runs before, but now I understand it _would_ always have changes, regardless.